app/enterprise/2.1.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -3,7 +3,10 @@ title: Kong for Kubernetes deployment options
 skip_read_time: true
 ---
 <div class="alert alert-ee blue">
-<b>Coming Soon:</b> Kong for Kubernetes Enterprise (K4K8s) is not currently available for use with {{site.ee_product_name}} 2.1.x beta. It will be released within the beta timeframe.
+For the {{site.ee_product_name}} 2.1.x beta, Kong for Kubernetes Enterprise
+(K4K8s) uses the <code>kong-enterprise-edition</code> image, which works as a
```

Comment:
Are we still keeping the same naming convention, i.e. K4K8s for DB-less and Kong Enterprise on Kubernetes for DB-backed?

Reply:
Per Reza's comment in DOCS-1166, effectively yes for now. It's not ideal but it will suffice for the beta; it's an area that we should expect user confusion on and we should revisit the naming questions before GA.